src/systemtask/SystemTask.cpp

```diff
@@ -344,8 +344,7 @@ void SystemTask::Work() {
         case Messages::OnNewHour:
           using Pinetime::Controllers::AlarmController;
           if (settingsController.GetNotificationStatus() != Controllers::Settings::Notification::Sleep &&
-              settingsController.GetChimeOption() == Controllers::Settings::ChimesOption::Hours &&
-              alarmController.State() != AlarmController::AlarmState::Alerting) {
+              settingsController.GetChimeOption() == Controllers::Settings::ChimesOption::Hours && alarmController.IsAlerting()) {
```

Comment:
```diff
- settingsController.GetChimeOption() == Controllers::Settings::ChimesOption::Hours && alarmController.IsAlerting()) {
+ settingsController.GetChimeOption() == Controllers::Settings::ChimesOption::Hours && !alarmController.IsAlerting()) {
```
and same for `HalfHours` below, or chimes won't work. LGTM apart from this

Related: For anyone that wants more chime frequency options / configurable chime duration, see my [chimes-ui-redesign](https://github.com/ght/InfiniTime/tree/chimes-ui-redesign) branch

---
@NeroBurner Thanks for adopting my abandoned PR! I mostly stopped coding, and am a farrier's apprentice  instead for almost two years now and much happier :)

Reply:
Indeed, no chimes currently on my watch :sweat_smile: Thanks. Fixed in rebase

You're welcome! And thanks for still coming back to us reviewing this rebase :)